### DISCUSSION TITLE
Duplicate an existing box ?

### DISCUSSION THREAD

**Discussion by minhbac260** [sandboxie-plus/Sandboxie#4299] [Q&A Feedback] [answered]
Hi everyone.

How can i duplicate an existing sandbox ?

I try some method but it not like what i want.

For example: I want to install Firefox and its extension on the first box, after that, i duplicate box and inside the second box, Firefox still has its extension.

I tried export the first box and import again to make the second box, but it does not work

I also try duplicate box config and copy all data from first box to second box, it does not work again.

Then, I tried snapshot manager, but snapshot is not what I want. I want to create a default box with essential apps and duplicate that box for different purposes (I not have to install essential apps each time create new box).

So, can anyone help me?

**Comment by offhub** [comment]
Sandboxie-Plus > Right click on the sandbox > Sandbox Options > Edit Ini Section > Copy and paste the line to the bottom of the list > OK

```ini
WriteFilePath=firefox.exe,*\AppData\**\Mozilla\*
```

See [WriteFilePath](https://sandboxie-plus.github.io/sandboxie-docs/Content/WriteFilePath/)

**Reply by minhbac260** [reply]
Thank you for replying to me.

I still don't understand all about WriteFilePath and how it works in this case.

Before adding the line you suggest, I have tried again and realize that if I run Firefox from C:\Program Files\Mozilla Firefox, it still has extensions, but from Run > Run from start menu > Desktop or Program, it does not

After I add the line you suggest, nothing have changed.

So, I think the problem is where I run Firefox.

**Reply by offhub** [reply]
Please share your sandboxie configuration. (Sandboxie-Plus > Options > Edit Sandboxie.ini > Copy/Paste)

> I still don't understand all about WriteFilePath and how it works in this case.

It hides folders outside the sandbox while allowing new ones to be created within it.

If you are using Firefox Portable, you need to change the profile path defined for WriteFilePath accordingly.

#### For default Firefox installation:
1. Create a new box (Do not enable any Firefox template!)
2. Add the rule
3. Run Firefox
4. Add your extensions
5. Export the box
6. From now on, you can use this exported box instead of creating a new box.

If you plan to install applications other than Firefox in this sandbox, it would be better to use a "Data Protection" type box. [***](https://sandboxie-plus.github.io/sandboxie-docs/PlusContent/privacy-mode/)

**Reply by minhbac260** [reply]
1. This is my config file in second box (also on the first box):

Enabled=y
BlockNetworkFiles=y
RecoverFolder=%Desktop%
RecoverFolder=%Personal%
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
BorderColor=#02f6f6,ttl
Template=AutoRecoverIgnore
Template=LingerPrograms
Template=BlockPorts
Template=qWave
Template=FileCopy
Template=SkipHook
Template=OpenBluetooth
ConfigLevel=10
UseFileDeleteV2=y
UseRegDeleteV2=y
AutoRecover=y
WriteFilePath=firefox.exe,*\AppData\**\Mozilla\*


2. In the first box, I run Firefox Installer.exe to install Firefox normally.

3. OK, Im now using Default Box, I will try "Data Protection" type box later

**Reply by minhbac260** [reply, marked as answer]
I don't know why but maybe this is a solution.

1. Create first box with essential apps and export it (any box)
2. Delete first box
3. Freely create more boxes with archive file of the first box

### **Just remember delete the first box.**

I guess there is a way redirect Firefox from new box to the first box. It causes my issue.
